aws/resource_aws_sagemaker_notebook_instance.go

```diff
@@ -85,6 +85,12 @@ func resourceAwsSagemakerNotebookInstance() *schema.Resource {
 				}, false),
 			},
 
+			"default_code_repository": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: false,
```

Comment:
no need to explicitly set `ForceNew: false`.

can you add update logic for this argument? or change the ForceNew value to true and this can done in a subsequent PR although its preferable to make this PR more complete

Reply:
Removed `ForceNew:false`

Reply:
as we dont have any way to update (not implemented in this PR) lets return the attribute and set it to true until update is implemented.

Reply:
Misunderstood the earlier comment. Added back ForceNew and set it to true. I will put out a follow up PR with the updated logic